roles/fstat/tasks/main.yml

```diff
@@ -31,3 +31,16 @@
     name: fstat
     enabled: True
     state: started
+
+- name: Configure Cron tab centos
+  cron:
+    name: fstat {{item.name}}
+    minute: "0"
+    hour: "1"
+    job: "/fstat/env/bin/python /fstat/code/manage.py process_jobs -n 3 -j {{item.job}}"
+
```

Comment:
I tested it on my local it didn't seem to have an issue, I checked the crontab file and it looked correct to me.